src/pages/account/mijn-materiaalaanvragen/index.tsx

```diff
@@ -119,6 +143,7 @@ const AccountMyMaterialRequests: NextPage<DefaultSeoInfo> = ({ url }) => {
 					{isFetching && <Loading owner="Material requests overview" />}
 					{noData && renderEmptyMessage()}
 					{!noData && !isFetching && renderContent()}
+					{renderDetailBlade()}
```

Comment:
Don't render the blade if there's no detail id / data available for the blade available.